test/README.md

```diff
@@ -57,6 +57,16 @@ go test -v -tags=e2e -count=1 ./test/conformance
 go test -v -tags=e2e -count=1 ./test/e2e
 ```
 
+## Running performance tests
+
+To run [the performance tests](./performance), you need to have a running environment 
+that meets [the test environment requirements](#environment-requirements), and you need
+to specify the build tag `performance`.
+
+```bash
+go test -v -tags=performance -count=1 ./test/performance
+```
+
 ### One test case
```

Comment:
can you update the title here to be "Running single test case" for the sake of consistency with above